.github/workflows/bump-version.yml

```diff
@@ -23,20 +23,26 @@ jobs:
         with:
           fetch-depth: 0
 
+      - uses: pnpm/action-setup@v2
+        name: Install pnpm
+        with:
+          version: 8
+          run_install: false
+
       - name: ⚙️ Setup Node.js
         uses: actions/setup-node@v2
         with:
-          node-version: '16'
-          cache: 'npm'
+          node-version: '18.x'
```

Comment:
Would suggest moving to node v20, as now 18 is just under maintenance

<img width="832" alt="image" src="https://github.com/ERC725Alliance/erc725.js/assets/31145285/38bd6440-130a-49bf-aceb-e3744e19be42">

Reply:
Switched to v20